contrib/golang/filters/http/source/go/pkg/http/asan.go

```diff
@@ -0,0 +1,34 @@
+package http
+
+import (
+	"runtime"
+	"sync"
+
+	"github.com/envoyproxy/envoy/contrib/golang/common/go/api"
+)
+
+var (
+	asanTestEnabled = false
+)
+
+// forceGCFinalizer enforce GC and wait GC finalizer finished.
+// Just for testing, to make asan happy.
+func forceGCFinalizer() {
+	var wg sync.WaitGroup
+	wg.Add(1)
+
+	{
+		// create a fake httpRequest to trigger GC finalizer.
+		fake := &httpRequest{}
+		runtime.SetFinalizer(fake, func(*httpRequest) {
+			wg.Done()
+		})
+	}
+
+	api.LogDebugf("golang filter enforcing GC")
```

Comment:
```suggestion
	api.LogWarn("golang filter enforcing GC")
```

I would recommend using a warn log so that when forceGCFinalizer is called in the code by accident, people can discover it.

Reply:
make sense, thanks